src/bin.ts

```diff
@@ -116,7 +120,8 @@ const service = register({
   ignoreDiagnostics: argv.ignoreDiagnostics,
   compilerOptions: parse(argv.compilerOptions) || DEFAULTS.compilerOptions,
   readFile: isEval ? readFileEval : undefined,
-  fileExists: isEval ? fileExistsEval : undefined
+  fileExists: isEval ? fileExistsEval : undefined,
+  transformers: argv.transformers ? require(join(cwd, argv.transformers)) : DEFAULTS.transformers
```

Comment:
This should use node module resolution instead of plain cwd join. This would enable the publishing of transformers to NPM. This should also support `argv.transformers` as an array and not just a string.